actions/pacstrap_action.go

```diff
@@ -42,24 +46,31 @@ SigLevel = Required DatabaseOptional TrustAll
 
 const configRepoSection = `
 
-[%s]
-Server = %s
+[%[1]s]
+Server = %[2]s
```

Comment:
Not sure if I mentioned this before - with a single server/mirror we can drop the explicit "Server =" for each repository and use `Include = /etc/pacman.d/mirrorlist` instead. That means that we'll need to (re)create it and write the server URL in there.